tests/countries/test_chile.py

```diff
@@ -204,9 +200,9 @@ def test_national_unity(self):
             "2002-09-02",
         )
 
-        self.assertNoHolidayName(
+        self.assertNoHolidayNameInYears(
             "Día de la Unidad Nacional",
-            Chile(years=set(range(1915, 2050)).difference({1999, 2000, 2001})),
+            tuple(set(range(1915, 2050)).difference({1999, 2000, 2001})),
```

Comment:
Looks heavy.
I'd just add set - `{list, set, tuple}` - here https://github.com/dr-prodigy/python-holidays/blob/beta/tests/common.py#L82

Reply:
Done. Thank you!